CONTRIBUTING.md

```diff
@@ -187,6 +187,47 @@ poetry run qa  # run all checks
 
 Ensure that your contribution follows the [PEP 8 – Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/) style guide with respect to patterns, naming, documentation and testing.
 
+## Issuing a New Release 
```

Comment:
Should this be on a different file. Not sure if this makes sense in CONTRIBUTING.md

Reply:
i'm not sure the standard way though, didn't find many examples. `tartufo` put them with contributing doc https://tartufo.readthedocs.io/en/latest/CONTRIBUTING.html, pytighub has a [DEPLOY.md](https://github.com/PyGithub/PyGithub/blob/master/DEPLOY.md). anyways, I moved them to `RELEASE.md` now